metricbeat/Makefile

```diff
@@ -19,11 +19,16 @@ GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 
 include ${ES_BEATS}/libbeat/scripts/Makefile
 
+.PHONY: clean
+clean::
+	# Clean generated dashbord directories
+	rm -rf _meta/kibana/dashboard _meta/kibana/search _meta/kibana/visualization
+	# Clean module docs
+	rm -rf docs/modules
+
 # Collects all module dashboards
 .PHONY: kibana
 kibana:
```

Comment:
I'm kind of undecided on this PR now looking at it again. It's nice that all clean is one place but at the same time it then requires kibana to depend on clean which I don't like. We should not get old dashboards because of CI always starts on a fresh build and would detect the old files.